execution_workflows/Roar/workflow/envs/APA-Scan.Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Template Dockerfile with python and pandas.
```

Comment:
File name includes APA-Scan. Also, it doesn't look like anything Roar-specific is being done in this Dockerfile